src/components/Staking/StakingCommunityCallout.tsx

```diff
@@ -29,7 +29,11 @@ const StyledButtonLink = styled(ButtonLink)`
   }
 `
 
-const StakingCommunityCallout = (props) => {
+export interface IProps {
+  id?: string
+}
+
+const StakingCommunityCallout: React.FC<IProps> = (props, id) => {
```

Comment:
`id` is coming inside the `props` arg.
```suggestion
const StakingCommunityCallout: React.FC<IProps> = (props) => {
```

Reply:
👍